bvm/ballerina-runtime/src/main/resources/MessagesBundle.properties

```diff
@@ -274,3 +274,4 @@ config.env.unused.vars = [{0}] unused environment variable
 config.env.variable.name.ambiguity = configurable environment variable ''{0}'' clashes for variable ''{1}'' with \
   variable ''{2}''
 no.worker.message.received = no message received from worker ''{0}'' to worker ''{1}''
+invalid.member.size = insufficient member size: expected at least ''{0}'', found ''{1}''
```

Comment:
Please add a new line at the end of this file.

Reply:
Thank you for pointing that out. Fixed with 717906f159d143fc9b5c9c2fb35042521b60f2a0